samples/client/petstore/haskell-http-client/lib/OpenAPIPetstore/MimeTypes.hs

```diff
@@ -223,33 +223,33 @@ instance MimeType MimeXmlCharsetutf8 where
 -- instance MimeRender MimeXmlCharsetutf8 T.Text where mimeRender _ = undefined
 -- instance MimeUnrender MimeXmlCharsetutf8 T.Text where mimeUnrender _ = undefined
 
--- ** MimeTextxml
+-- ** MimeTextXml
 
-data MimeTextxml = MimeTextxml deriving (P.Typeable)
```

Comment:
@jonschoning a heads up that this change will update the naming of Haskell variables, e.g. MimeTextxml => MimeTextXml (capital letter X).

Please let me know if you've concerns with this change.

Reply:
I think this is fine

Reply:
👌 I'll mark this as a breaking change to inform Haskell users of the breaking changes (change in variable naming)